public/app/features/dimensions/editors/ResourcePickerPopover.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { useEffect, useState } from 'react';
+import { css } from '@emotion/css';
+import SVG from 'react-inlinesvg';
+import { Button, Select, FilterInput, useTheme2, stylesFactory, Field, Label, Input } from '@grafana/ui';
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { FocusScope } from '@react-aria/focus';
+
+import { ResourceCards } from './ResourceCards';
+import { getPublicOrAbsoluteUrl } from '../resource';
+import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
+import { FileElement, GrafanaDatasource } from 'app/plugins/datasource/grafana/datasource';
+import { ResourceFolderName } from '..';
+
+interface Props {
+  value?: string; //img/icons/unicons/0-plus.svg
+  onChange: (value?: string) => void;
+  mediaType: 'icon' | 'image';
```

Comment:
Can you turn this typoing into an enum? I see it's being used in a few places in a similar way.